CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+
+## [4.13.0]
+
+### Added
+
+- Added support for password reset for users that exist only on BC (not yet synced with WP)
+- Delete all channels on plugin uninstall so fresh channel list is pulled when reinstalling and connecting to a new store
+- Added support for changing the SKU element when selecting variants. This adds a wrapper 
+  around the SKU value where JS targets the wrapper and updates the text.
+  
+  **NOTE:** Update your templates accordingly. *product-sku.php*
```

Comment:
The template version was bumped up meaning that now it will show up in diagnostic reports as an outdated tempalte. This note is for developers to check if they modified that template and then for them to update it to bring it in line with the new version.